src/Rector/Psr4/MultipleClassFileToPsr4ClassesRector.php

```diff
@@ -214,7 +214,7 @@ private function processNodesWithoutNamespace(array $nodes, SmartFileInfo $smart
             $fileDestination = $this->createClassLikeFileDestination($node, $smartFileInfo);
 
             if ($declareNode) {
-                $nodes = array_merge([$declareNode], [$node]);
+                $nodes = [$declareNode, $node];
```

Comment:
Good one, let's make a Rector from this:
https://github.com/rectorphp/rector/issues/2170